website/docs/r/storagegateway_gateway_activation_key.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "aws"
+page_title: "AWS: aws_storagegateway_gateway_activation_key"
+sidebar_current: "docs-aws-resource-storagegateway-gateway-activation-key"
+description: |-
+  Provides a resource to retrieve an activation key from a gateway using the region of the provider
+---
+
+# aws_storagegateway_gateway_activation_key
+
+Provides a resource to retrieve an activation key from a gateway using the region of the provider.
+
+~> **NOTE:** Terraform must be able to make an HTTP (port 80) GET request to the specified IP address from where it is running.
+
+~> **NOTE:** Deleting this Terraform resource does not perform any operations.
```

Comment:
then should this be a data source?

Reply:
Re-implemented as data source after I figured out that data sources also support configurable timeouts 😄